tests/h/search/core_test.py

```diff
@@ -243,3 +245,17 @@ def test_only_200_replies_are_included(self, pyramid_request, Annotation):
 
         assert len(result.reply_ids) == 3
         assert oldest_reply.id not in result.reply_ids
+
+
+@pytest.fixture(params=[True, False])
```

Comment:
Since a string representation of the param value is used in generated test names, I think it would be easier to understand what is happening when looking at the `--verbose` output or any test failures if the params were string labels (eg. `es1`, `es6`) rather than booleans.

As it is, the output currently looks like this:

```
tests/h/search/query_test.py::TestUsersAggregation::test_it_returns_annotation_counts_by_user[True] PASSED                    [ 96%]
tests/h/search/query_test.py::TestUsersAggregation::test_it_returns_annotation_counts_by_user[False] PASSED                   [ 97%]
tests/h/search/query_test.py::TestUsersAggregation::test_it_limits_number_of_annotation_counts_by_user_returned[True] PASSED  [ 98%]
tests/h/search/query_test.py::TestUsersAggregation::test_it_limits_number_of_annotation_counts_by_user_returned[False] PASSED [100%]
```